Composer/packages/client/src/pages/publish/Publish.tsx

```diff
@@ -86,6 +86,17 @@ const Publish: React.FC<RouteComponentProps<{ projectId: string; targetName?: st
     [projectId, publishTypes]
   );
 
+  const isPublishDisable = useMemo(() => {
+    // if current profile is already publishing, set disable to true
+    if (selectedTargetName && selectedTargetName !== 'all') {
+      if (thisPublishHistory?.length >= 0) {
```

Comment:
when this ```if``` is not true, nothing is returned, do you want it to be undefined?

Reply:
fixed